cmd/dfget/app/version.go

```diff
@@ -24,17 +24,33 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// versionDescription is used to describe version command in detail and auto generate command doc.
+var versionDescription = "Display the version and build information of Dragonfly dfget， " +
+	"including GoVersion, Version, BuildDate, GitBranch and GitCommit."
+
 var versionCmd = &cobra.Command{
 	Use:           "version",
-	Short:         "Show the current version",
+	Short:         "Show the current version of dfget",
+	Long:          versionDescription,
 	SilenceErrors: true,
 	SilenceUsage:  true,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		fmt.Printf("%s\n", version.DFGetVersion)
+		fmt.Println(version.Print("dfget"))
 		return nil
 	},
+	Example: versionExample(),
 }
 
 func init() {
 	rootCmd.AddCommand(versionCmd)
 }
+
+// versionExample shows examples in version command, and is used in auto-generated cli docs.
+func versionExample() string {
+	return `$ dfget, version  0.4.1-14-g61aa7a9
```

Comment:
remove the character `$` and string `-14-g61aa7a9`.
Usually a release is built on a tag, and the version value is the tag name without its prefix `v`.

Reply:
Done